metrics/integration.go

```diff
@@ -41,6 +42,14 @@ var (
 		},
 	)
 
+	SnapshotCreatedToPipelineRunStartedSeconds = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "integration_svc_snapshot_created_to_pipelinerun_with_env_started_seconds",
```

Comment:
Suggestion to make the name even simpler:
```suggestion
			Name:    "integration_svc_snapshot_created_to_pipelinerun_started_seconds",
```

Reply:
Done